classifier/nic.go

```diff
@@ -27,5 +30,54 @@ func checkNICs(profile *hwcc.HardwareClassification, host *bmh.BareMetalHost) bo
 		"ok", ok,
 	)
 
+	if !ok {
+		return ok
```

Comment:
You are returning bool so always return true or false.

Reply:
Okay, addressed.